### ISSUE TITLE
Results about GA

### ISSUE THREAD

**Issue by guangjh** [Z1zs/MMUnlearner#3]
Thank you for your work. However, when I tried to reproduce the GA method, I didn't make any changes to your code, but the results were all 0, and the generated output was a bunch of gibberish. I tried both zero-shot and few-shot approaches, but both gave me 0. I'm not sure why.
{
    "Forget Set Results": {
        "fill_in_the_blank": {
            "image_textual_accuracy": 0.0,
            "pure_text_accuracy": 0.0
        },
        "classification": {
            "Image-Textual Question Accuracy": 0.0,
            "Pure Text Question Accuracy": 0.0
        },
        "generation": {
            "Average ROUGE-1 (Image_Textual)": 0.0,
            "Average ROUGE-2 (Image_Textual)": 0.0,
            "Average ROUGE-L (Image_Textual)": 0.0,
            "Average BLEU (Image_Textual)": 0.0,
            "Average ROUGE-1 (Pure_Text)": 0.0,
            "Average ROUGE-2 (Pure_Text)": 0.0,
            "Average ROUGE-L (Pure_Text)": 0.0,
            "Average BLEU (Pure_Text)": 0.0
        }
    }
}

**Comment by Z1zs** [comment]
Hi @guangjh 

Thanks a lot for your interest at our work! Based on my experience, GA is very unstable and may result in a idiot model after unlearning. To obtain reasonable results, you may try to:

1. Decrease learning_rate
2. Set smaller epochs

Please tell me if such settings can solve your problem!

**Comment by guangjh** [comment]
In my experiment，I set：
        --forget_split_ratio=5 \
	--batch_size=2 \
	--lr=1e-5 \
	--num_epochs=1 
I get a idiot model after unlearning,I'm not sure why.

**Comment by Z1zs** [comment]
Hi @guangjh , which dataset you use for your experiment? As far as I know the quality of CLEAR is not very good, therefore you may get wired results on it. Similar phenomena has been reported in the original paper(https://arxiv.org/abs/2410.18057).

**Comment by guangjh** [comment]
I use MLLMU dataset

**Comment by Z1zs** [comment]
It seems wired. Have you met similar question on other models or datasets? Or could you plz try to print the loss during GA?
